### PR TITLE
Ensure offline videos are accessible on native devices

### PR DESCRIPTION
**Pull request by tonytlwu**
Ref. https://github.com/Fliplet/fliplet-studio/issues/5660

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
   }
 
   Fliplet.Widget.instance('video-offline', function (data) {
-    var url = _.get(data, 'file.video.bundledFile.url');
+    var url = _.get(data, 'file.video.bundledFile.url') || _.get(data, 'file.video.selectFiles.url');
```
Comment (squallstar) [inline js/build.js:19]: Why is this change necessary?

Comment (tonytlwu) [reply js/build.js:19]: Because if `bundledFile.url` isn't available, for old widget instances, then the video source will be based on the `selectFiles.url` property. That would need to be checked and authenticated if necessary.

**Review by squallstar — APPROVED** [review]
